apps/kitchen-sink/src/features/Menu/assets/subitem_icon.svg

```diff
@@ -0,0 +1,2 @@
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 88 88"><defs><style>.cls-1{fill:none;}.cls-2{fill:#00000000;fill-rule:evenodd;stroke: white;stroke-width:1.5px}</style></defs><title>subtask store</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><rect class="cls-1" width="88" height="88" rx="8"/><path class="cls-2" d="M23,21a4,4,0,0,0-4,4v2a4,4,0,0,0,4,4h2V62a1,1,0,0,0,1,1h9a4,4,0,0,0,4,4H65a4,4,0,0,0,4-4V61a4,4,0,0,0-4-4H39a4,4,0,0,0-4,4H27V45h8a4,4,0,0,0,4,4H65a4,4,0,0,0,4-4V43a4,4,0,0,0-4-4H39a4,4,0,0,0-4,4H27V31H65a4,4,0,0,0,4-4V25a4,4,0,0,0-4-4Z"/></g></g></svg>
```

Comment:
Why not import it from https://github.com/mondaycom/monday-ui-style/tree/master/src/Icons ?